src/app/app.component.ts

```diff
@@ -537,7 +538,10 @@ export class AppComponent implements OnInit {
   getInterpolationType(curveType) {
     return this.curves[curveType] || this.curves['default'];
   }
-
+  isTensionSliderRequired(curveType) {
+    return this.curves[curveType] == this.curves['Cardinal'] ||
+    this.curves[curveType] == this.curves['Catmull Rom'];
+  }
   setColorScheme(name) {
```

Comment:
New line between functions to match coding style.
```suggestion
  }
  
  setColorScheme(name) {
```